tests/cases/fourslash/completionsWithOptionalPropertiesGeneric.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path="fourslash.ts" />
+// @strict: true
+
+//// interface MyOptions {
+////     hello?: boolean;
+////     world?: boolean;
+//// }
+//// declare function bar<T extends MyOptions>(options?: Partial<T>): void;
```

Comment:
mind adding a case with unconstrained `T`? It should have no completions I think, but I just want to make sure.